src/app/shared/tabview-x/tabview-x.component.ts

```diff
@@ -144,7 +144,7 @@ export class TabPanelExtendedComponent extends TabPanel implements AfterContentI
      </div>
   `,
   styleUrls: [
-    './tabview.scss',
+    './tabview.css',
```

Comment:
is this a bug fix?

Reply:
It actually is. It compiled just fine before. But the file is named `tabview.css`, not `tabview.scss`.